src/test/java/com/yegor256/rpm/RpmITCase.java

```diff
@@ -105,10 +105,16 @@ public void savesAndLoads() throws Exception {
                     "cat /repo/repodata/primary.xml",
                     "cat /repo/repodata/filelists.xml",
                     "cat /repo/repodata/other.xml",
-                    "dnf config-manager --verbose --add-repo file:///repo",
+                    "echo '[repo]' >> /etc/yum.repos.d/repo.repo",
+                    "echo 'name=test-repo' >> /etc/yum.repos.d/repo.repo",
+                    "echo 'gpgcheck=0' >> /etc/yum.repos.d/repo.repo",
+                    "echo 'enabled=1' >> /etc/yum.repos.d/repo.repo",
+                    "echo 'baseurl=file:///repo' >> /etc/yum.repos.d/repo.repo",
```

Comment:
@Sammers21 this is dirty. use `--setopt=<option>=<value>` instead: https://dnf-plugins-core.readthedocs.io/en/latest/config_manager.html

Reply:
I've tried and it does not work for me. Let's keep it the way it is now.

Yes, it's dirty, but it's more maintainable and easy to read

Reply:
@Sammers21 what exactly `dnf config-manager` produces? Don't trust Artifactory too much, their team is far from being perfect

Reply:
```
# dnf config-manager --add-repo=file:///repo123 --setopt=gpgcheck=0
# cat /etc/yum.repos.d/repo123.repo
[repo123]
name=created by dnf config-manager from file:///repo123
baseurl=file:///repo123
enabled=1
```
@yegor256 , as you can see, gpgcheck=0 property is not written to the file

Reply:
@Sammers21 maybe this will help: https://bugzilla.redhat.com/show_bug.cgi?id=1253237

Reply:
Asked a question about it on SO: https://stackoverflow.com/questions/59486727/why-dnf-config-manager-save-does-not-work

Reply:
@yegor256 , would it be fine if we would generate `.repo` file with `dnf config-manager` and append `gpgcheck=0` to this file manually?

Reply:
@Sammers21 OK, let's do it this way